src/libraries/OSGroups.json

```diff
@@ -38,6 +38,28 @@
         ]
       },
     "NetBSD": {
+      "#import": [
+        "Unix"
+        ]
+      },
+    "Illumos": {
+      "#import": [
+        "Unix"
+        ]
+      }
+    },
+    "Solaris": {
+      "#import": [
+        "Unix"
+        ]
+      },
+    "Illumos": {
+      "#import": [
+        "Unix"
+        ]
+      }
+    },
+    "Solaris": {
```

Comment:
@ericstj, for some reason, these groups were generated twice. Maybe because I ran with `/p:UpdateRuntimeFiles=true` twice? Should it add these it idempotently?

Reply:
The purpose of this file is to define values which can be used in TargetFramework strings.

As far as I know, this file isn't auto-generated.  It only exists because we haven't done the unification to make our config-system use the real runtime graph. That's issue https://github.com/dotnet/runtime/issues/32451

cc @Anipik

Reply:
I must have made a (scp) copy/paste error here from VM to localhost before committing the changes. Removed the duplicate along with capitalization change.